docs/root/operations/certificates.rst

```diff
@@ -0,0 +1,13 @@
+.. _operations_certificates:
+
+Certificate Management
+======================
+
+Envoy provides several mechanisms for cert management. At a high level they can be broken into
+
+1. Static :ref:`CommonTlsContext <envoy_api_msg_auth.CommonTlsContext>` referenced certificates.
+   These will *not* reload automatically, and requires either a restart of the proxy or
```

Comment:
nit: you may want to distinguish between hot restart and regular restart i.e. hot restart can be triggered when they change